.github/workflows/ci.yml

```diff
@@ -133,39 +133,13 @@ jobs:
           components: rust-src
       - uses: Swatinem/rust-cache@v2
 
-      # Verify the MSRV for all RISC-V chips.
+      # Verify the MSRV for all chips by running a lint session
       - name: msrv RISCV (esp-hal)
         run: |
-          cargo xtask build-package --features=esp32c2,ci --target=riscv32imc-unknown-none-elf   esp-hal
-          cargo xtask build-package --features=esp32c3,ci --target=riscv32imc-unknown-none-elf   esp-hal
-          cargo xtask build-package --features=esp32c6,ci --target=riscv32imac-unknown-none-elf  esp-hal
-          cargo xtask build-package --features=esp32h2,ci --target=riscv32imac-unknown-none-elf  esp-hal
-
-      - name: msrv RISCV (esp-wifi)
-        run: |
-          cargo xtask build-package --features=esp32c2,wifi,ble,esp-hal/unstable --target=riscv32imc-unknown-none-elf   esp-wifi
-          cargo xtask build-package --features=esp32c3,wifi,ble,esp-hal/unstable --target=riscv32imc-unknown-none-elf   esp-wifi
-          cargo xtask build-package --features=esp32c6,wifi,ble,esp-hal/unstable --target=riscv32imac-unknown-none-elf  esp-wifi
-          cargo xtask build-package --features=esp32h2,ble,esp-hal/unstable --target=riscv32imac-unknown-none-elf  esp-wifi
-
-        # Verify the MSRV for all Xtensa chips:
+          cargo +stable xtask lint-packages --chips esp32c2,esp32c3,esp32c6,esp32h2
```

Comment:
Is this using the correct version? I think this should be `+${{ env.MSRV }}` or something.

Reply:
I think it was fine, but I've updated to to use the MSRV toolchain explicitly

Reply:
Eh, this didn't work out, but I added extra steps to print the toolchain versions before we use it.